test/k4FWCoreTest/src/components/ExampleFunctionalConsumerRuntimeCollectionsMultiple.cpp

```diff
@@ -70,12 +70,10 @@ struct ExampleFunctionalConsumerRuntimeCollectionsMultiple final
       fatal() << "Wrong size of the tracks map, expected 3, got " << trackMap.size() << endmsg;
     }
     for (auto& [key, tracks] : trackMap) {
-      if ((tracks[0].getType() != 1) || (std::abs(tracks[0].getChi2() - 2.1) > 1e-6) || (tracks[0].getNdf() != 3) ||
-          (std::abs(tracks[0].getRadiusOfInnermostHit() - 6.1) > 1e-6)) {
+      if ((tracks[0].getType() != 1) || (std::abs(tracks[0].getChi2() - 2.1) > 1e-6) || (tracks[0].getNdf() != 3)) {
         std::stringstream error;
         error << "Wrong data in tracks collection, expected 1, 2.1, 3, 4.1, 5.1, 6.1 got " << tracks[0].getType()
-              << ", " << tracks[0].getChi2() << ", " << tracks[0].getNdf() << ", "
-              << tracks[0].getRadiusOfInnermostHit() << "";
+              << ", " << tracks[0].getChi2() << ", " << tracks[0].getNdf() << "";
```

Comment:
```suggestion
              << ", " << tracks[0].getChi2() << ", " << tracks[0].getNdf();
```

Reply:
I noticed and then forgot because I wanted to look at the whole repo... There were a few more occurrences but they should all be gone now.